mobile_rewards/src/server.rs

```diff
@@ -52,7 +55,7 @@ impl Server {
         let result = Self {
             pool,
             keypair,
-            follower_service: FollowerService::from_env()?,
+            follower_service: FollowerService::from_env().expect("follower service env error"),
```

Comment:
ditto

Reply:
Fixed in [656f9f4](https://github.com/helium/oracles/pull/115/commits/656f9f4d1984816f0f4cb7069a9bed68b81d25c9)